diff --git a/TeXmacs/progs/texmacs/keyboard/prefix-kbd.scm b/TeXmacs/progs/texmacs/keyboard/prefix-kbd.scm
index e34abe5..283f5c2 100644
--- a/TeXmacs/progs/texmacs/keyboard/prefix-kbd.scm
+++ b/TeXmacs/progs/texmacs/keyboard/prefix-kbd.scm
@@ -86,10 +86,6 @@
   ("structured:geometry" "M-C-"))
 
 (kbd-wildcards
-  ("escape" "M-" #t)
-  ("escape escape" "A-" #t)
-  ("escape escape escape" "C-" #t)
-  ("escape escape escape escape" "noop" #t)
   ("S-escape" "M-C-" #t)
   ("S-escape S-escape" "A-C-" #t)
   ("S-escape S-escape S-escape" "M-A-" #t)
